<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1027</width>
    <height>690</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>.AppleSystemUIFont</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #2D2D2D;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>991</width>
      <height>621</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page1">
     <widget class="QPushButton" name="helpButton">
      <property name="geometry">
       <rect>
        <x>689</x>
        <y>490</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: white;
    color: #2D2D2D;
    border-radius: 17px; 
	font-family: &quot;Century Gothic&quot;, sans-serif;
	font-size: 20px;
}




</string>
      </property>
      <property name="text">
       <string>?</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>310</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    background: transparent;
    color: gray;
	font-size: 11px;
 font-family: &quot;Century Gothic&quot;, sans-serif;
}
</string>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loginButton">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>460</y>
        <width>261</width>
        <height>32</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #57B9FF;
    color: white;
    border-radius: 15px;
    font-family: &quot;Century Gothic&quot;, sans-serif;
}
</string>
      </property>
      <property name="text">
       <string>Log In</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>86</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Impact</family>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    background-color: none;
    color: white;
    border: 0px solid gray;
    border-radius: 10px;
    padding: 5px;
    font-size: 40px;
font-family: &quot;Impact&quot;, sans-serif;
    font-weight: bold;
}</string>
      </property>
      <property name="text">
       <string>NAS APP</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>250</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    background: transparent;
    color: gray;
	font-size: 11px;
 font-family: &quot;Century Gothic&quot;, sans-serif;
}
</string>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>160</y>
        <width>341</width>
        <height>371</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
    background-color: white; 
    border: 0px solid lightgray; 
    border-radius: 50px; 
    padding: 10px; 
}
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QLineEdit" name="NameLineEdit">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>110</y>
         <width>261</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
background-color: white; 
color: black; 
border: 2px solid lightgray;
border-radius: 15px; 
padding: 5px;

}



</string>
       </property>
      </widget>
      <widget class="QPushButton" name="forgotButton">
       <property name="geometry">
        <rect>
         <x>193</x>
         <y>264</y>
         <width>121</width>
         <height>32</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: none; 
    color: grey; 
	font-size: 10px;
    border-radius: 15px; 
 font-family: &quot;Century Gothic&quot;, sans-serif;

}
</string>
       </property>
       <property name="text">
        <string>Forgot Password</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>30</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
    background-color: none;
    color:  #2D2D2D;
    border: 0px solid gray;
    border-radius: 10px;
    padding: 5px;
    font-size: 25px;
 font-family: &quot;Century Gothic&quot;, sans-serif;
    font-weight: bold;
}
</string>
       </property>
       <property name="text">
        <string>Welcome</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="usernameLineEdit">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>170</y>
         <width>261</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
background-color: white; 
color: black; 
border: 2px solid lightgray;
border-radius: 15px; 
padding: 5px;
}

</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="passwordLineEdit">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>230</y>
         <width>261</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
background-color: white; 
color: black; 
border: 2px solid lightgray;
border-radius: 15px; 
padding: 5px;
}

</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::EchoMode::Password</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="signupPageButton">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>330</y>
         <width>161</width>
         <height>32</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: white;
    color: #57B9FF;
    border-radius: 15px;
    font-family: &quot;Century Gothic&quot;, sans-serif;
}
</string>
       </property>
       <property name="text">
        <string>Sign Up</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>370</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    background: transparent;
    color: gray;
	font-size: 11px;
 font-family: &quot;Century Gothic&quot;, sans-serif;
}
</string>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_31">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>90</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Impact</family>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    background-color: none;
    color: #57B9FF;;
    border: 0px solid gray;
    border-radius: 10px;
    padding: 5px;
    font-size: 40px;
font-family: &quot;Impact&quot;, sans-serif;
    font-weight: bold;
}</string>
      </property>
      <property name="text">
       <string>NAS APP</string>
      </property>
     </widget>
     <zorder>label_31</zorder>
     <zorder>frame</zorder>
     <zorder>helpButton</zorder>
     <zorder>label</zorder>
     <zorder>loginButton</zorder>
     <zorder>label_6</zorder>
     <zorder>label_3</zorder>
     <zorder>label_2</zorder>
    </widget>
    <widget class="QWidget" name="page3">
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>220</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    background-color: none;
    color:  white;
    border: 0px solid gray;
    border-radius: 10px;
    padding: 5px;
    font-size: 25px;
 font-family: &quot;Century Gothic&quot;, sans-serif;
    font-weight: bold;
}
</string>
      </property>
      <property name="text">
       <string>Log in Successful</string>
      </property>
     </widget>
     <widget class="QPushButton" name="backButton">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>370</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: white;
    color: #2D2D2D;
    border-radius: 17px; 
	font-family: &quot;Century Gothic&quot;, sans-serif;
	font-size: 15px;
}




</string>
      </property>
      <property name="text">
       <string>Back to Log In</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>392</x>
        <y>270</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    background-color: none;
    color:  white;
    border: 0px solid gray;
    border-radius: 10px;
    padding: 5px;
    font-size: 15px;
 font-family: &quot;Century Gothic&quot;, sans-serif;
    font-weight: bold;
}
</string>
      </property>
      <property name="text">
       <string>&lt;connect to home page&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page2">
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>110</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    background-color: none;
    color:  white;
    border: 0px solid gray;
    border-radius: 10px;
    padding: 5px;
    font-size: 25px;
 font-family: &quot;Century Gothic&quot;, sans-serif;
    font-weight: bold;
}
</string>
      </property>
      <property name="text">
       <string>Help page</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>130</y>
        <width>731</width>
        <height>131</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    background-color: none;
    color:  white;
    border: 0px solid gray;
    border-radius: 10px;
    padding: 5px;
    font-size: 15px;
 font-family: &quot;Century Gothic&quot;, sans-serif;
    font-weight: bold;
}
</string>
      </property>
      <property name="text">
       <string>This project creates a local file storage system using a Raspberry Pi with a</string>
      </property>
     </widget>
     <widget class="QPushButton" name="backButton_2">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>440</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: white;
    color: #2D2D2D;
    border-radius: 17px; 
	font-family: &quot;Century Gothic&quot;, sans-serif;
	font-size: 15px;
}




</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>160</y>
        <width>581</width>
        <height>131</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    background-color: none;
    color:  white;
    border: 0px solid gray;
    border-radius: 10px;
    padding: 5px;
    font-size: 15px;
 font-family: &quot;Century Gothic&quot;, sans-serif;
    font-weight: bold;
}
</string>
      </property>
      <property name="text">
       <string>desktop app for uploading, downloading, and managing files securely.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>190</y>
        <width>581</width>
        <height>131</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    background-color: none;
    color:  white;
    border: 0px solid gray;
    border-radius: 10px;
    padding: 5px;
    font-size: 15px;
 font-family: &quot;Century Gothic&quot;, sans-serif;
    font-weight: bold;
}
</string>
      </property>
      <property name="text">
       <string>Upload Files: Click &quot;Upload&quot; to transfer files onto the NAS from your computer.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_37">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>220</y>
        <width>601</width>
        <height>131</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    background-color: none;
    color:  white;
    border: 0px solid gray;
    border-radius: 10px;
    padding: 5px;
    font-size: 15px;
 font-family: &quot;Century Gothic&quot;, sans-serif;
    font-weight: bold;
}
</string>
      </property>
      <property name="text">
       <string>Find Files: Use the search bar or the categories, such as &quot;Images&quot;, to find files fast.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_38">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>250</y>
        <width>601</width>
        <height>131</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    background-color: none;
    color:  white;
    border: 0px solid gray;
    border-radius: 10px;
    padding: 5px;
    font-size: 15px;
 font-family: &quot;Century Gothic&quot;, sans-serif;
    font-weight: bold;
}
</string>
      </property>
      <property name="text">
       <string>Select multiple files: Select multiple files to download, delete, or move at once.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_59">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>280</y>
        <width>601</width>
        <height>131</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    background-color: none;
    color:  white;
    border: 0px solid gray;
    border-radius: 10px;
    padding: 5px;
    font-size: 15px;
 font-family: &quot;Century Gothic&quot;, sans-serif;
    font-weight: bold;
}
</string>
      </property>
      <property name="text">
       <string>File Sorting: Click &quot;Sort&quot; to change the file orders by name, date, or file type.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_60">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>310</y>
        <width>601</width>
        <height>131</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    background-color: none;
    color:  white;
    border: 0px solid gray;
    border-radius: 10px;
    padding: 5px;
    font-size: 15px;
 font-family: &quot;Century Gothic&quot;, sans-serif;
    font-weight: bold;
}
</string>
      </property>
      <property name="text">
       <string>Download: Click &quot;Download&quot; to save files from NAS to your computer.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_61">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>390</y>
        <width>601</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    background-color: none;
    color:  white;
    border: 0px solid gray;
    border-radius: 10px;
    padding: 5px;
    font-size: 15px;
 font-family: &quot;Century Gothic&quot;, sans-serif;
    font-weight: bold;
}
</string>
      </property>
      <property name="text">
       <string>Files are stored securely on NAS with encryption, so only you can access them.</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page4">
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>180</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    background-color: none;
    color:  white;
    border: 0px solid gray;
    border-radius: 10px;
    padding: 5px;
    font-size: 25px;
 font-family: &quot;Century Gothic&quot;, sans-serif;
    font-weight: bold;
}
</string>
      </property>
      <property name="text">
       <string>Forgot Password</string>
      </property>
     </widget>
     <widget class="QPushButton" name="backButton_3">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>380</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: white;
    color: #2D2D2D;
    border-radius: 17px; 
	font-family: &quot;Century Gothic&quot;, sans-serif;
	font-size: 15px;
}




</string>
      </property>
      <property name="text">
       <string>Back to Log In</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>250</y>
        <width>431</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    background-color: none;
    color:  white;
    border: 0px solid gray;
    border-radius: 10px;
    padding: 5px;
    font-size: 15px;
 font-family: &quot;Century Gothic&quot;, sans-serif;
    font-weight: bold;
}
</string>
      </property>
      <property name="text">
       <string>Enter your username and we will retrieve your password:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="NameLineEdit_2">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>310</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
background-color: white; 
color: black; 
border: 2px solid lightgray;
border-radius: 15px; 
padding: 5px;

}



</string>
      </property>
     </widget>
     <widget class="QPushButton" name="retrieveButton">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>380</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: white;
    color: #2D2D2D;
    border-radius: 17px; 
	font-family: &quot;Century Gothic&quot;, sans-serif;
	font-size: 15px;
}




</string>
      </property>
      <property name="text">
       <string>Retrieve Password</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page5">
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>190</y>
        <width>341</width>
        <height>351</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
    background-color: white; 
    border: 0px solid lightgray; 
    border-radius: 50px; 
    padding: 10px; 
}
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QLineEdit" name="NameLineEdit_signup">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>110</y>
         <width>261</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
background-color: white; 
color: black; 
border: 2px solid lightgray;
border-radius: 15px; 
padding: 5px;

}



</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_27">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>20</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
    background-color: none;
    color:  #2D2D2D;
    border: 0px solid gray;
    border-radius: 10px;
    padding: 5px;
    font-size: 25px;
 font-family: &quot;Century Gothic&quot;, sans-serif;
 
}
</string>
       </property>
       <property name="text">
        <string>Sign Up</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="usernameLineEdit_signup">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>170</y>
         <width>261</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
background-color: white; 
color: black; 
border: 2px solid lightgray;
border-radius: 15px; 
padding: 5px;
}

</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="passwordLineEdit_signup">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>230</y>
         <width>261</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
background-color: white; 
color: black; 
border: 2px solid lightgray;
border-radius: 15px; 
padding: 5px;
}

</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::EchoMode::Password</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="signupButton">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>280</y>
         <width>261</width>
         <height>32</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #57B9FF;
    color: white;
    border-radius: 15px;
    font-family: &quot;Century Gothic&quot;, sans-serif;
}
</string>
       </property>
       <property name="text">
        <string>Sign Up</string>
       </property>
      </widget>
      <widget class="QPushButton" name="backButton_7">
       <property name="geometry">
        <rect>
         <x>98</x>
         <y>320</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: white;
    color: grey;
    border-radius: 17px; 
	font-family: &quot;Century Gothic&quot;, sans-serif;
	font-size: 10px;
}




</string>
       </property>
       <property name="text">
        <string>Back to Log In</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>280</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    background: transparent;
    color: gray;
	font-size: 11px;
 font-family: &quot;Century Gothic&quot;, sans-serif;
}
</string>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_29">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>340</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    background: transparent;
    color: gray;
	font-size: 11px;
 font-family: &quot;Century Gothic&quot;, sans-serif;
}
</string>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_30">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>400</y>
        <width>50</width>
        <height>10</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    background: transparent;
    color: gray;
	font-size: 11px;
 font-family: &quot;Century Gothic&quot;, sans-serif;
}
</string>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1027</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
